src/app/settings/components/project-role-jwt-tokens/project-role-jwt-tokens.tsx

```diff
@@ -25,7 +25,7 @@ export const ProjectRoleJWTTokens = (props: ProjectRoleJWTTokensProps) => {
             {(ctx) => (
                 <React.Fragment>
                 <h4>JWT Tokens</h4>
-                {props.tokens !== null && props.tokens.length > 0 ? (
```

Comment:
It should be `props.tokens && props.tokens.length > 0 `. Otherwise linter is going to complain